pkg/slayers/path/scion/decoded_spec.gobra

```diff
@@ -98,6 +99,13 @@ func (d *Decoded) IncPath(ghost ubuf []byte) (e error) {
 	return e
 }
 
+ghost
+requires d.Mem(ubuf)
+decreases
+pure func (d *Decoded) PathCurrINFValid (ghost ubuf []byte) (bool) {
+	return unfolding d.Mem(ubuf) in unfolding d.Base.Mem() in 0 <= d.PathMeta.CurrINF && d.PathMeta.CurrINF < 3
```

Comment:
why don't we split this in more lines to make it easier to read?
```suggestion
	return unfolding d.Mem(ubuf) in
	    unfolding d.Base.Mem() in
	    0 <= d.PathMeta.CurrINF && d.PathMeta.CurrINF < 3
```
An idiom that I've been using as of late is that for long pure expressions, I put every `unfolding` and `let` expression in a separate line, treating the keyword `in` as an indicator for a new line. this way, we can read this as if it was a sequence of stmts